demo/danesfield_demo.py

```diff
@@ -0,0 +1,134 @@
+import os
+import sys
+import subprocess
+import boto3
+from collections import defaultdict
+import argparse
+
+def run(args):
+	parser = argparse.ArgumentParser(description="Download WV3 images to run the Danesfield pipeline")
+	parser.add_argument('--site', help='CORE3D site to download images for', \
+						default='Jacksonville')
+	parser.add_argument('--img_path', help='Location where WV3 images should be downloaded to', \
+						default=f'{os.getcwd()}/imgpath')
+	parser.add_argument('--run', help='Run Danesfield after downloading images', \
+						action='store_true')
+	parser.add_argument('--out_path', help='Location that Danesfield should use as an output directory', default='outdir')
```

Comment:
Watch the line lengths, as code formatting guidelines suggest max 80 characters per line.